pkg/velero/restore.go

```diff
@@ -0,0 +1,32 @@
+package velero
+
+import (
+	"sort"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetRestoreNamespace return the namespace mapping for the given namespace
+func GetRestoreNamespace(ns, bkpName string, log logrus.FieldLogger) (string, error) {
+	listOpts := metav1.ListOptions{}
+	list, err := clientSet.VeleroV1().Restores(veleroNs).List(listOpts)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to get list of restore")
+	}
+
+	sort.Sort(sort.Reverse(RestoreByCreationTimestamp(list.Items)))
```

Comment:
Can you comment above this line to indicate why sort was used?
- This logic seems to work because only one restore is active on a given backup name. Is this true. 
- Also is it not possible to pass the restore id or something to this function. If not, specify in the comment that velero doesn't send the restore id, so we have to depend on the backup name and the order of invocation. And at any given time there will ideally be only one RestorePhaseInProgress for a given backup name.

Reply:
ok. I'll add the comment similar to PR description.